cspatients/util.py

```diff
@@ -127,6 +127,8 @@ def split_patient_and_entry_data(data):
     patient_data = {}
```

Comment:
@erikh360 Would you mind adding a docstring here? Something like:
```
Splits the data from a RapidPro POST request

@param data: <some description of data>
```
I'm not even sure it's accurate though 😅